sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderer.kt

```diff
@@ -60,14 +60,48 @@ internal abstract class MapboxRenderer : MapClient {
 
   @AnyThread
   override fun scheduleRepaint() {
-    renderThread.requestRender()
+    Logger.e("KIRYLDD", "scheduleRepaint!")
+    renderThread.queueRenderEvent(
+      RenderEvent(
+        runnable = null,
+        needRender = true,
+        eventType = EventType.SDK
+      )
+    )
   }
 
   @AnyThread
   override fun scheduleTask(task: Task) {
-    renderThread.queueEvent {
-      task.run()
-    }
+    Logger.e("KIRYLDD", "scheduleTask!")
```

Comment:
maybe change tag? :)

Reply:
I'll take another look at it tomorrow on other devices and then remove 😄 
Will mark PR as draft for now